app/src/main/kotlin/com/hippo/ehviewer/ui/screen/DownloadsScreen.kt

```diff
@@ -168,8 +175,12 @@ fun DownloadsScreen(navigator: DestinationsNavigator) = composing(navigator) {
     val labelEmpty = stringResource(R.string.label_text_is_empty)
     val defaultInvalid = stringResource(R.string.label_text_is_invalid)
     val labelExists = stringResource(R.string.label_text_exist)
-    val downloadsCount by rememberInVM { EhDB.downloadsCount }.collectAsState(emptyMap())
-    val totalCount = remember(downloadsCount) { downloadsCount.values.sum() }
+    val downloadsCountGroupByArtist = rememberInVM(invalidateKey) {
```

Comment:
Why we need `rememberInVM` not `remember`?
`downloadInfoList` may change after user navigate to other destinations so this may be outdated